package.json

```diff
@@ -106,22 +106,22 @@
     "@antfu/eslint-config": "^0.25.2",
     "@types/fs-extra": "^9.0.13",
     "@types/html-minifier": "^4.0.2",
-    "@types/jsdom": "^16.2.14",
-    "@types/prettier": "^2.6.3",
-    "@types/yargs": "^17.0.10",
-    "@vueuse/head": "^0.7.6",
+    "@types/jsdom": "^20.0.0",
+    "@types/prettier": "^2.7.0",
+    "@types/yargs": "^17.0.11",
+    "@vueuse/head": "^0.7.9",
     "bumpp": "^8.2.1",
     "critters": "^0.0.16",
-    "eslint": "^8.18.0",
+    "eslint": "^8.21.0",
     "esno": "^0.16.3",
-    "rollup": "^2.75.7",
-    "tsup": "^6.1.2",
+    "rollup": "^2.77.2",
+    "tsup": "^6.2.1",
     "typescript": "^4.7.4",
-    "unbuild": "^0.7.4",
-    "vite": "^2.9.13",
-    "vite-plugin-pwa": "^0.12.1",
```

Comment:
Also here.

Reply:
I think it is enough to only change the dependency of the route. I restored other dependent versions